workflow/tests/test_other.rb

```diff
@@ -98,42 +98,54 @@ def test_run_simulation_faster_performance
     assert_equal(0, component_loads.size)
   end
 
-  def test_run_simulation_detailed_occupancy_schedules
-    [false, true].each do |debug|
-      # Check that the simulation produces stochastic schedules if requested
-      sample_files_path = File.join(File.dirname(__FILE__), '..', 'sample_files')
-      tmp_hpxml_path = File.join(sample_files_path, 'tmp.xml')
-      hpxml = HPXML.new(hpxml_path: File.join(sample_files_path, 'base.xml'))
-      XMLHelper.write_file(hpxml.to_doc, tmp_hpxml_path)
-
-      rb_path = File.join(File.dirname(__FILE__), '..', 'run_simulation.rb')
-      xml = File.absolute_path(tmp_hpxml_path)
-      command = "\"#{OpenStudio.getOpenStudioCLI}\" \"#{rb_path}\" -x \"#{xml}\" --add-stochastic-schedules"
-      command += ' -d' if debug
-      system(command, err: File::NULL)
+  def test_run_simulation_stochastic_occupancy_schedules
```

Comment:
It looks like a lot of changes here, but all I'm really doing is:
1. Looping through a few HPXML files, not just base.xml, to test a variety of HPXML objects with simple schedules
2. Check run.log has no warnings about both simple and detailed schedules (before these changes, all of the files would)